README.md

```diff
@@ -15,6 +15,12 @@ composer require rapidez/image-resizer
 ```
 And make sure you ran `php artisan storage:link`
 
+## Image from SKU
+
+You can retrieve a product image by using the product's SKU. This is enabled by default, but can be toggled with the `allow_sku` value in the config file.
+
+To retrieve a product image using SKU, request a path like this: `/storage/resizes/200x200/sku/13706`. You can also request a webp like this: `/storage/resizes/200x200/sku/13706?webp`.
```

Comment:
So when `webp` is wanted it doesn't match with the storage path and it's still going through php?

Reply:
For either case it will go through the php once, generating a permanent redirect to the correct image url.

Reply:
So for every new visitor it's going through PHP to redirect. Maybe Cloudflare caches the redirect but better to match the file just "normal" images work? Create symlinks so the images aren't stored twice?

Reply:
I'm just wondering now what'll happen if the image of a product gets changed around. Surely the permanent redirect and symlink will then both permanently point towards a dead image?

Reply:
Redirect will fail or point to an old image if it's still present but not linked to the product anymore. But if Nginx can't find the image it will go through PHP which gives the option to change the symlink.